pipeline/hard-reset.sh

```diff
@@ -90,46 +91,46 @@ if [ "${force_yes}" = false ] ;then
 fi
 
 ######### functions ###########
-hard_reset_shard
+function hard_reset_shard
 {
    shard=$1
-   msg "cat $logdir/shard${shard}.txt | xargs -i{} -P50 bash -c \"./restart_node.sh -U -X -D -y -d logs/${net_profile} -p ${net_profile} -t 0 -r 0 -R 0 {}\""
-   cat "$logdir/shard${shard}.txt" | xargs -i{} -P50 bash -c "./restart_node.sh -U -X -D -y -d logs/${net_profile} -p ${net_profile} -t 0 -r 0 -R 0 {}"
+   msg "cat $logdir/shard${shard}.txt | xargs -i% -P50 bash -c \"./restart_node.sh -U -X -D -y -d logs/${net_profile} -p ${net_profile} -t 0 -r 0 -R 0 %\""
+   cat "$logdir/shard${shard}.txt" | xargs -i% -P50 bash -c "./restart_node.sh -U -X -D -y -d logs/${net_profile} -p ${net_profile} -t 0 -r 0 -R 0 %"
 }
 
-check_consensus
+function check_consensus
 {
    shard=$1
    msg "./run_on_shard.sh -p ${net_profile} -T $shard 'tac /home/tmp_log/*/zeorlog*.log | grep -m 1 HOORAY'"
```

Comment:
This is the part just to print out the cmd to be run, need one more line to run the actual cmd.

Reply:
Also need to add `-y` to avoid waiting. Otherwise, it will be stuck there.

Reply:
The above 2 comments apply to the functions below I believe.

Reply:
will enable in the next PR.